packages/paste-core/components/button-group/src/ButtonGroup.tsx

```diff
@@ -49,7 +49,20 @@ const UnattachedStyled = styled.span(
 );
 
 export interface ButtonGroupProps extends Omit<React.ComponentPropsWithRef<"div">, "children"> {
```

Comment:
This needs to extend `HTMLPasteProps<'div'>`, not `Omit<React.ComponentPropsWithRef<"div">, "children">`.

I'm not sure where that pattern came from but I've updated the plop templates, and any component in my batch that is using it.